code/game/objects/items/storage/fancy.dm

```diff
@@ -29,8 +29,7 @@
 
 /obj/item/storage/fancy/update_icon(itemremoved = 0)
 	if(fancy_open)
-		var/total_contents = src.contents.len - itemremoved
-		icon_state = "[icon_type]box[total_contents]"
+		icon_state = "[icon_type]box[src.contents.len]"

```

Comment:
Remove the src

Reply:
Should I also delete the other src. like over here:
to_chat(user, "There are [contents.len <= 0 ? "no" : "[src.contents.len]"] [src.icon_type]s left.")

Reply:
for content.len

Reply:
sure if you want